Trellix/client/http_client.py

```diff
@@ -158,31 +189,248 @@ def epo_events_url(self, start_date: datetime, limit: int = 10) -> URL:
 
         return URL("{0}/epo/v2/events".format(self.base_url)).with_query(params)
 
-    async def get_epo_events(
-        self, start_date: datetime, limit: int = 10
-    ) -> List[TrellixEdrResponse[EpoEventAttributes]]:
+    def edr_threats_url(self, start_date: datetime, end_date: datetime, limit: int = 10, offset: int = 0) -> URL:
+        """
+        Get EDR threats url.
+
+        Filter by `from` should be enough to get threats for specified window,
+            as it applies on `lastDetected` field.
+
+        It will get all threats from start_date to now.
+
+        Doc:
+        https://developer.manage.trellix.com/mvision/apis/threats
+
+        Args:
+            start_date: datetime
+            end_date: datetime
+            limit: int
+            offset: int
+
+        Returns:
+            URL:
+        """
+        params: dict[str, int | str] = {
+            "from": int(start_date.strftime("%s")) * 1000,
+            "to": int(end_date.strftime("%s")) * 1000,
```

Comment:
Why you don't use `.timestamp()`?
```suggestion
            "from": int(start_date.timestamp()) * 1000,
            "to": int(end_date.timestamp()) * 1000,
```